mozilla/blurts-server
# Load the dashboard into an actual page

There is still a bunch of hardcoded data, and it doesn't show data broker exposures yet, but at least it's a more realistic setting - it already exposed a couple of places where things didn't work perfectly (e.g. components with a transparent background, or data in a different shape than expected).

Note that unfortunately, I wasn't able to re-use the Storybook story in the unit test, because the import of `getL10n` would trigger an error because that imports `headers` from `next/headers`, which in turn triggers an error about not running on the server side. We could investigate that later, or possibly Next.js updates its Jest support to work better, or we could just not re-use the Storybook and its mock data.

## How to test

Visit `/redesign/user/dashboard` while logged in - you should see `<ExposureCard>`s listing your breaches, albeit all with the Twitter logo for now, and with every breach listed as "Fixed". Also, I've updated the Storybook to remove the mock "Shell" story (you can see what the Shell looks like when looking at the Dashboard or, later, the Settings), and the old non-[page](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts) dashboard, and instead added [a story](https://deploy-preview-3209--fx-monitor-storybook.netlify.app/?path=/story/pages-dashboard--dashboard) that renders the actual page (with mock data).
